docs/content/schedules/_index.md

```diff
@@ -10,16 +10,21 @@ resticprofile is capable of managing scheduled backups for you. Under the hood i
 - **launchd** on macOS X
 - **Task Scheduler** on Windows
 - **systemd** where available (Linux and other BSDs)
-- **crond** on supported platforms (Linux and other BSDs)
+- **crond** as fallback (depends on the availability of a `crontab` binary)
+- **crontab** files (low level)
 
-On unixes (except macOS) resticprofile is using **systemd** by default. **crond** can be used instead if configured in `global` `scheduler` parameter:
+On unixes (except macOS) resticprofile is using **systemd** if available and falls back to **crond**. 
+On any OS a **crond** compatible scheduler can be used instead if configured in `global` / `scheduler`:
 
 {{< tabs groupid="config-with-json" >}}
 {{% tab title="toml" %}}
 
 ```toml
 [global]
   scheduler = "crond"
+  # scheduler = "crond:/usr/bin/crontab"
+  # scheduler = "crontab:*:/etc/cron.d/resticprofile"
+  # scheduler = "crontab:-:/var/spool/cron/crontabs/username"
```

Comment:
we should probably have a small section explaining the parameters, specially the middle one 🤔

Reply:
Or a link to the reference / global section as it is mentioned there. However was also thinking it may be needed.

Reply:
A link to the reference would probably do 😄

Reply:
added both 🙂